include/swift/AST/SemanticAttrs.def

```diff
@@ -0,0 +1,79 @@
+//===--- Semantics.def - Semantics Attribute Definitions -------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This macro is used to map the global definition name of a semantic
```

Comment:
Please follow LLVM style here:

https://llvm.org/docs/CodingStandards.html#file-headers

Notice how this part of the file header has a \file and also has 3 slashes.